np asks for OTP when we run npm run release

I tried to run `np` as in the README.  It asked for a OTP, but I didn't have one.  I tried pressing the button on my security key, and it failed.

```
Command failed with exit code 1: npm publish --tag alpha --otp

...

npm notice Publishing to https://registry.npmjs.org/ with tag alpha and default access
npm ERR! code E400
npm ERR! 400 Bad Request - PUT https://registry.npmjs.org/lux-design-system - child "otp" fails because ["otp" with value "[REDACTED]" fails to match the required pattern: /^\d+$/, "otp" length must be 64 characters long]

npm ERR! A complete log of this run can be found in: /Users/sandbergj/.npm/_logs/2023-12-28T18_59_59_952Z-debug-0.log
```

--- a/README.md
+++ b/README.md
@@ -21,6 +21,10 @@ npm install --global np # don't add this as a devDependency, otherwise you won't
 npm run release
 ```
 
+#### Tips
+* If you have a passphrase on your ssh key, run ssh-add ~/.ssh/id_ed25519 (or wherever your key is located).
+* If you use a security key, rather than an authenticator app, for two-factor authentication in npmjs.com, make sure that "Require two-factor authentication for write actions" is not checked in your account 2FA settings. If it is checked, np will ask you for an OTP from your phone, and won't allow you to push without it.
+
 ### Deploy the styleguide to GitHub Pages
 ```
 npm run deploy

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "test": "jest",
     "lint": "vue-cli-service lint",
     "deploy": "npm run build; npm run styleguide:build; push-dir --dir=styleguide --branch=gh-pages --cleanup",
-    "release": "np --no-publish && npm publish && npm run deploy",
+    "release": "npm run build && np && npm run deploy",
     "styleguide": "vue-cli-service styleguidist",
     "styleguide:build": "vue-cli-service styleguidist:build"
   },

